pages/index.js

```diff
@@ -3,7 +3,17 @@ import Image from 'next/image';
 
 import styles from '@/styles/Home.module.scss';
 
+import useUser from 'lib/useUser';
+
 export default function Home() {
+  const { user } = useUser({
+    redirectTo: '/login',
+  });
```

Comment:
Not sure whether this works or not, I visited the home screen without logging in but there's no redirect and the `Loading` message is still there 💭 
![Screen Shot 2565-04-05 at 16 10 17](https://user-images.githubusercontent.com/1772999/161720788-461dd733-43df-4fd1-9119-75259c8d0e63.png)

Reply:
that's interesting, even after setting up the `env`?

Reply:
Ahh, I retried and found that it's because I have the local storage with the `Authentication` key 😂  once remove it now it works fine 🙌🏼